multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperUtil.java

```diff
@@ -27,14 +27,18 @@ public class MapperUtil {
 
   public static final String BIG_DECIMAL = "bigDecimal";
 
+  public static final String REF = "$ref";
+
+  private static final String DIVISOR = "(\\.|\\/)";
+
   private MapperUtil() {}
 
   public static String getSimpleType(final JsonNode schema, final String prefix, final String suffix) {
     String type = schema.textValue();
     if (schema.has("type")) {
       type = schema.get("type").textValue();
       String format = null;
-      if (type.equalsIgnoreCase("string")) {
+      if ("string".equalsIgnoreCase(type)) {
```

Comment:
Not an issue with this PR in particular, but the AsyncAPI code would probably benefit from a similar type refactor as the one we did on OpenAPI

Reply:
This change it's not related with the development, I made the change due to I found it with yhe `CheckStyle` or the `SonarlLint` (the "boyscout" rule).
A comment that we have also made, it's I would like to make a refactor to reduce the code complexity.
I have corrected what I have seen where I have touched, but this is an issue from an user and I prefer to solve it fast, and male the big changes later.